ccd-config-generator/src/test/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -135,6 +137,7 @@
     @NotNull(message = "You need to add details to hearing needed")
     @Valid
     private final Hearing hearing;
+    @JsonUnwrapped(prefix = "hearingPreferences")
```

Comment:
I guess you could default the prefix to the field name

Reply:
I decided to stick to the default behaviour of JsonUnwrapped